scheduler/src/cook/mesos/api.clj

```diff
@@ -2209,6 +2235,14 @@
                    :handler (read-jobs-handler-multiple conn framework-id is-authorized-fn
                                                         retrieve-sandbox-directory-from-agent)}}))
 
+        (c-api/context
+          "/info" []
+          (c-api/resource
+            {:get {:summary "Returns info about this Cook scheduler instance's setup. No authentication required."
+                   :responses {200 {:schema CookInfo
+                                    :description "The Cook scheduler info was returned."}}
```

Comment:
Nitpick: I think we should capitalize Scheduler in both places

Reply:
Done